js/adapt-contrib-confidenceSlider.js

```diff
@@ -93,33 +97,45 @@ define([
         },
 
         _setupLinkedModel: function() {
-            var linkedModel = Adapt.components.findWhere({_id: this.model.get('_linkedToId')});
-            this.model.set({
-                '_showNumber':linkedModel.get('_showNumber'),
-                '_showScaleIndicator':linkedModel.get('_showScaleIndicator'),
-                '_showScale':linkedModel.get('_showScale'),
-                'labelStart':linkedModel.get('labelStart'),
-                'labelEnd':linkedModel.get('labelEnd'),
-                '_scaleStart':linkedModel.get('_scaleStart'),
-                '_scaleEnd':linkedModel.get('_scaleEnd')
-            });
-            this.model.set('_linkedModel', linkedModel);
-            if (this.model.get('_attempts') < 0) linkedModel.set('_attempts', 1);
+            try {
+                var linkedModel = Adapt.components.findWhere({_id: this.model.get('_linkedToId')});
+
```

Comment:
Instead of the try catch will this work?
```js
if (!linkedModel) { 
  Adapt.log.error("Please check that you have set _linkedToId correctly!");
  return;
}
```

Reply:
fine by me... amended, could you re-review?